vaadin-tabs-flow-parent/vaadin-tabs-flow/src/test/java/com/vaadin/flow/component/tabs/tests/TabTest.java

```diff
@@ -59,4 +61,25 @@ public void shouldSetFlexGrow() throws Exception {
     public void implementsHasTooltip() {
         assertTrue(tab instanceof HasTooltip);
     }
+
+    @Test
+    public void implementHasAriaLabel() {
+        assertTrue(tab instanceof HasAriaLabel);
+    }
+
+    @Test
+    public void setAriaLabel() {
+        tab.setAriaLabel("aria label");
+
+        Assert.assertTrue(tab.getAriaLabel().isPresent());
+        Assert.assertEquals("aria label", tab.getAriaLabel().get());
```

Comment:
Would it make sense to set `aria-label` empty in the tests like with the `MenuItemBase` [tests](https://github.com/vaadin/flow-components/pull/4771/files#diff-1b31e108fe04a1e2b36adf97cdea69857910ce7bb5d4d2f0b71420a92e31017cR70-R71)?

Reply:
Done.